pkg/kudoctl/cmd/init/manager.go

```diff
@@ -0,0 +1,218 @@
+package init
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/version"
+
+	"k8s.io/api/apps/v1beta2"
+	v1 "k8s.io/api/core/v1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	clientv1beta2 "k8s.io/client-go/kubernetes/typed/apps/v1beta2"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"sigs.k8s.io/yaml"
+)
+
+//Defines the deployment of the KUDO manager and it's service definition.
+
+const (
+	group              = "kudo.dev"
+	crdVersion         = "v1alpha1"
+	defaultns          = "kudo-system"
+	defaultGracePeriod = 10
+)
+
+// Options is the configurable options to init
+type Options struct {
+	// Version is the version of the manager `0.5.0` for example (must NOT include the `v` in `v0.5.0`)
+	Version string
+	// Namespace to init into (default is kudo-system)
+	Namespace string
+	// TerminationGracePeriodSeconds defines the termination grace period for a pod
+	TerminationGracePeriodSeconds int64
+	// Image defines the image to be used
+	Image string
+}
+
+// NewOptions provides an option struct with defaults
+func NewOptions(v string) Options {
```

Comment:
you could use *string to represent Option<string>

Reply:
this

Reply:
I'm not sure if I understand your comment here.. if it is to change `v string` to `v *string`... looking into that it complicates things it ways that are less readable IMO.  It seems fine.